examples/vad_transcriber/requirements.txt

```diff
@@ -0,0 +1,4 @@
+deepspeech==0.2.0
```

Comment:
@b-ak Are you sure you want to force a version?

Reply:
The models won't with with older versions right ?

Reply:
And some of the code changes are also not compatible with older versions

Reply:
So it'd be best to use matching master?

Reply:
Yes, I wrote the whole thing on top of the 0.2.0. I will test it again on top of the master when I rebase it. That'll be a good confirmation right ?

Reply:
Whichever you feel is the best, we could go that way :)

Reply:
@b-ak Rebase it and we merge, you can enhance after